mailbox/plugin/deleted-messages-vault/build.gradle

```diff
@@ -2,27 +2,64 @@
  * This file was generated by the Gradle 'init' task.
```

Comment:
A detail: but we need a license header?

Reply:
My focus is to build the project.
Once we have that we can take care of the rest of the tasks.
If we can't build the project or it does not build faster, there is no point to do the rest of the work.

It can be done quite fast via intellij and string processing for all gradle files.
When the time comes. 
Current goal: get community buy-in for gradle. 
Prove Gradle brings benefits and commit to adopting it.